src/Compilers/CSharp/Portable/Symbols/Source/LambdaSymbol.cs

```diff
@@ -371,8 +371,9 @@ internal override void AddDeclarationDiagnostics(BindingDiagnosticBag diagnostic
                 var name = unboundLambda.ParameterName(p);
                 var location = unboundLambda.ParameterLocation(p);
                 var locations = location == null ? ImmutableArray<Location>.Empty : ImmutableArray.Create<Location>(location);
+                var isParams = paramSyntax?.Modifiers.Any(static m => m.Kind() == SyntaxKind.ParamsKeyword) ?? false;
```

Comment:
Nit: we have an `IsKind` method.

Reply:
Right, thanks. I saw uses of `Kind() == ...` everywhere, that's why I used it. Anyway, changed: 61a90b7e273290a9429c2a4a1fc719cecdb7939c